packages/docs/site/i18n/gu/docusaurus-plugin-content-docs/current.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
Some of those texts are used on the Docusaurus UI. You can translate the `message` property to Gujarati